sources/postgres/infoschema_test.go

```diff
@@ -445,15 +432,14 @@ func TestConvertSqlRow_MultiCol(t *testing.T) {
 			query: "SELECT (.+) FROM pg_index (.+)",
 			args:  []driver.Value{"public", "test"},
 			cols:  []string{"index_name", "column_name", "column_position", "is_unique", "order"},
-		},
-		// Note: go-sqlmock mocks specify an ordered sequence
```

Comment:
Did not follow why this is removed ?

Reply:
This was a redundant call and there was a TODO to remove this. This removal was also needed for dynamodb refactoring. I think I resolved 2-3 TODOs with this PR. Would it be better to create several small PRs for these multiple changes?

Reply:
Different PR would be good considering the size of this one. But i think this one should be fine